src/app.ts

```diff
@@ -58,12 +59,25 @@ async function app() {
 
   logger.info(`Starting ${getVersion()}`);
 
+  inDir = getInputDir(inDir);
+
   // Load dependencies
   const defs = new fhirdefs.FHIRDefinitions();
-  const dependencies = program.dependency;
-  const dependencyDefs = loadExternalDependencies(defs, dependencies);
+  let dependencies = program.dependency;
+
+  // Trim empty spaces from commad line dependencies
+  dependencies = dependencies?.map((dep: string) => dep.trim());
```

Comment:
Now that I'm seeing this together, I think you might as well simplify even further to:
```suggestion
  // Trim empty spaces from commad line dependencies
  const dependencies = program.dependency?.map((dep: string) => dep.trim());
```